app/src/main/java/ar/com/wolox/android/cookbook/common/di/AppModule.kt

```diff
@@ -49,4 +51,10 @@ abstract class AppModule {
 
     @ContributesAndroidInjector
     abstract fun instagramLoginFragment(): InstagramLoginRecipeFragment
+
+    @ContributesAndroidInjector
+    abstract fun roomActivity(): RoomRecipeActivity
```

Comment:
This is not wrong. AppModule is getting biggerr. We need to create a module for every case. I mean TwiiterModule, InstagramModule, etc.